network/containerizer/bridgepolicy.go

```diff
@@ -15,9 +15,11 @@ import (
 
 	"github.com/juju/juju/core/instance"
 	corenetwork "github.com/juju/juju/core/network"
+	"github.com/juju/juju/environs"
 	"github.com/juju/juju/network"
 
-	// Used for some constants and things like LinkLayerDevice[Args]
+	// TODO (manadart 2019-09-25): This package should evolve
+	// so as not to import this package.
```

Comment:
This would be great. State shouldn't leak out of package/state.

Reply:
Unfortunately the shims need to wrap state objects still.
We might not be able to do it...